app/views/exchanges/hbx_profiles/_confirm_create_ba_modal.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="modal" id="create_ba_mod" tabindex="-1" role="dialog" role="document">
```

Comment:
Confirm Modal added to additionally confirm the action for some specific cases during new Benefit Application Creation.